person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
```

Comment:
- [ ] **Optional**:  Hi, I would advise you to declare these values before the `initialize method`. This is logic used with ruby code. and this will make your code readable, and professional.

Reply:
thanks alot dear @OLIPLICHE .
i apply that and then I merge this PR